css-positioning/absolute.css

```diff
@@ -0,0 +1,47 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 15px;
+  top: 20px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 160px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
+  /* your code here */
+
+  position: absolute;
+  top: 50px;
+  bottom: 0;
+  width: 100%;
```

Comment:
You don't need to apply a `width` here. You can either use width / height OR you can use offset properties, not both.

So this should look like `top: 0`, `right: 0`, `bottom: 0`, `left: 0`. If you wanted to use width & height, you would remove the offset properties and just do `width: 300px` & `height: 300px`